### ISSUE TITLE
Bug: 10.0.0 Beta 2 Errors on Load

### ISSUE THREAD

**Issue by CDAGaming** [SFX-WoW/Masque#257]
### Game Flavor

Retail

### Game Version

10.0.0

### Add-On Version

10.0.0-Beta-2

### Description

See Error Messages

### Error Message

```text
2x FrameXML\Bindings.xml:1 Interface/AddOns/Masque/Core/Regions/Regions.xml:9 Error loading Interface/AddOns/Masque/SlotIcon.lua
```
```
2x Interface/AddOns/Masque/Core/Regions/Regions.xml:9 Error loading Interface/AddOns/Masque/SlotIcon.lua
```
```


### Steps to Reproduce

1. Load Game with Beta 2
2. Should see error

### Screenshots

_No response_

### Notes

_No response_

**Comment by MetalHeadSeven** [comment]
Same 2 errors for me.

**Comment by StormFX** [comment]
Fixed in [Beta-3](Getting errors with Beta-2). Sorry about that. I was tired and hungry. :P

**Comment by MetalHeadSeven** [comment]
No problem.  :)

**Comment by StormFX** [comment]
Fixed in 85cfd8d.
